toast/toast.css

```diff
@@ -0,0 +1,39 @@
+@import url("../styles.css");
+
+.baseline-btn {
+  background-color: var(--primary-dark);
+  border: none;
+  border-radius: 5px;
+  color: var(--font);
+  cursor: pointer;
+  font-size: 1.1rem;
+  font-weight: 600;
```

Comment:
make a local variable for font-weight in the main CSS file

Reply:
done